### PR TITLE
Jira issue: WP-9

### PR DESCRIPTION
**Pull request by eelcocramer**
Check if optional parameter 'parent' is set.

### CODE REVIEW

File: webinos/common/rpc/lib/registry.js [hunk 1]
```diff
@@ -67,7 +67,7 @@
 	Registry.prototype.registerObject = function (callback) {
 		_registerObject.call(this, callback);
 
-		if (this.parent.registerServicesWithPzh) {
+		if (this.parent !== undefined && this.parent.registerServicesWithPzh) {
```
Comment (aldafu) [inline webinos/common/rpc/lib/registry.js]: please just check for parent:

if (this.parent && this.parent.registerServicesWithPzh) ...

Comment (eelcocramer) [inline webinos/common/rpc/lib/registry.js]: Sure, will do.

**Comment by ghost** [comment]
The commit messages should contain the summary on the first line, an empty line and then the rest. So the first line should not be the jira issue reference. I'll show you in a minute and can re-arrange it for you very quickly then.

**Comment by eelcocramer** [comment]
I reverted the change and commited the change again with the correct format. When you pull this in you will probably get the reverted commits as well. Don't know for sure. Is there any way to get rid of them.

On a side note: is it possible to scan for the Jira issue anywhere in the commit message? This would make commenting more easy I guess. Just scan for 'WP-XXX' and change that into a Jira link.

**Comment by aldafu** [comment]
Usually in this situation you would have used git commit --amend to rewrite your previous commit without the need to revert it. At this point it might be the easiest for you to send a new pull request though and close this one here.

**Pull request closed without merging**